projects/core/src/error-handling/error-handling.module.ts

```diff
@@ -6,20 +6,16 @@
 
 import { ErrorHandler, ModuleWithProviders, NgModule } from '@angular/core';
 import { CxErrorHandler } from './cx-error-handler';
-import { ERROR_INTERCEPTORS } from './error-interceptors/error-interceptor';
-import { LoggerErrorInterceptor } from './error-interceptors/logger-error.interceptor';
+import { provideMultiErrorHandlers } from './multi-error-handlers';
 
-@NgModule()
+@NgModule({
+  providers: [provideMultiErrorHandlers()],
```

Comment:
Shouldn't we put it to `forRoot()` method, for consistency?

Reply:
Agreed!